department-of-veterans-affairs/caseflow
# Justin/APPEALS-60000

Resolves [Seed Data for Veterans to have inactive appeals](https://jira.devops.va.gov/browse/APPEALS-60000)

# Description
Writing seed data to add six inactive appeals to each veteran. Added 3 with a root task status of closed, and then 3 with a root task status of cancelled. 

## Acceptance Criteria
- [x] Caseflow Developer is to create three inactive appeals with a cancelled root task for every correspondence veteran in the demo environment
- [x] Caseflow Developer is to create three inactive appeals with a completed root task for every correspondence veteran in the demo environment 

## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below

1. Log into Caseflow as an inbound ops team user. 
2. Navigate to queue/correspondence. (Note: Admins and Superusers will have access to all the correspondence already, but regular iops team users may have to have a correspondence assigned to them before they will see them).
3. Select a veteran from the unassigned tab to go to the review package page
4. If necessary, add a correspondence type, and click save.
5. Scroll to the bottom and select "Create record"
6. Navigate to step 2
7. Find the "Tasks related to an existing Appeal" section and click the "yes" radio button
8. Note that the first six appeals should be inactive appeals. The first three will be assigned to "Mail" (cancelled), and the next three will be assigned to "Post-decision" (closed).
9. Choose one of these appeals and verify that it is inactive. You can verify this by following the next three steps.
10. Scroll down and click the "+ Add tasks" button
11. Click the task dropdown.
12. Verify that there are only 12 options in this list. A quick way to check is to make sure the list does not contain "Change Of Address", as this is only available on active appeals
13. Verify that the existing seed data (appeals AFTER the first 6) are still active. You can verify this by following the next three steps.
14. Scroll down and click the "+ Add tasks" button
15. Click the task dropdown.
16. Verify that there are more than 12 options in this list. A quick way to check is to make sure the list contains "Change Of Address", as this is only available on active appeals.
